vue-element-admin/public/.htaccess

```diff
@@ -0,0 +1,8 @@
+<ifModule mod_rewrite.c>
+    RewriteEngine On
+  RewriteBase /
+  RewriteRule ^index\.html$ - [L]
+  RewriteCond %{REQUEST_FILENAME} !-f
+  RewriteCond %{REQUEST_FILENAME} !-d
+  RewriteRule . /index.html [L]
+</ifModule>
```

Comment:
.htaccess doesn't work here, but you can follow step 3 in the readme: https://github.com/pixiebrix/playground#adding-a-project-with-a-build

Which means you can just duplicate this line to achieve the same:

https://github.com/pixiebrix/playground/blob/703ce4b5e209a26b2eca851b4575398d69414d36/vercel.json#L6-L9